src/tests/test_integration.py

```diff
@@ -48,10 +55,23 @@ def write_config(self, prefix='', name='tox.ini', **kwargs):
         if not os.path.isdir(base):
             self.makedirs(base)
 
+        name = self.config_name if name is None else name
+        if name.endswith('.toml'):
+            def convert_value(val):
+                if isinstance(val, bool):
+                    return {True: 'true', False: 'false'}[val]
+                else:
+                    return repr(val)
```

Comment:
```suggestion
                return repr(val).lower()
```
This can be made simpler too

Reply:
This will also convert `"D100"` to `"d100"` which is wrong. Although I **can** change the `dict` lookup in the first branch to `repr(val).lower()` in the first branch only.

Reply:
Interesting, I forgot about that. Perhaps

```py
return repr(val) if not isinstance(val, bool) else repr(val).lower()
```
would suffice?